### PR TITLE
[minigraph.py] Enable telemetry service by default

### PR DESCRIPTION
**Pull request by pra-moh**
**- What I did**
Enabling telemetry feature by default
**- How I did it**
on minigraph load adding new table in config db as 'features|telemetry' {'status':'on'}
**- How to verify it**
config load_minigraph
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -714,8 +714,10 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['DHCP_SERVER'] = dict((item, {}) for item in dhcp_servers)
     results['NTP_SERVER'] = dict((item, {}) for item in ntp_servers)
     results['TACPLUS_SERVER'] = dict((item, {'priority': '1', 'tcp_port': '49'}) for item in tacacs_servers)
-
     results['ACL_TABLE'] = acls
+    results['FEATURES'] = {'telemetry': {
+        'status': 'on'
+        }}
```
Comment (jleveque) [inline src/sonic-config-engine/minigraph.py]: If you want to make the definition multi-line (which will probably be best for readability if we add other services), I suggest the following format:

```
results['FEATURES'] = {
    'telemetry': {
        'status': 'on'
    }
}
```

Comment (pra-moh) [reply src/sonic-config-engine/minigraph.py]: makes sense. Updated

File: src/sonic-config-engine/minigraph.py [hunk 2]
```diff
@@ -728,8 +728,12 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['DHCP_SERVER'] = dict((item, {}) for item in dhcp_servers)
     results['NTP_SERVER'] = dict((item, {}) for item in ntp_servers)
     results['TACPLUS_SERVER'] = dict((item, {'priority': '1', 'tcp_port': '49'}) for item in tacacs_servers)
-
     results['ACL_TABLE'] = acls
+    results['FEATURES'] = {
+        'telemetry': {
+            'status': 'on'
```
Comment (jleveque) [inline src/sonic-config-engine/minigraph.py]: I think the values should be "enabled"/"disabled" instead of "on"/"off"

**Comment by jleveque** [comment]
Retest vs please

File: src/sonic-config-engine/minigraph.py (hunk 2 above)
Comment (nikos-github) [reply src/sonic-config-engine/minigraph.py]: It's not acceptable to enable this service by default.

**Comment by jleveque** [comment]
@nikos-github: Even when loading from minigraph? This section of code will only write this section to the config_db.json when loading from minigraph. Is this still an issue for you?

**Comment by nikos-github** [comment]
@jleveque Ok if this default enabling doesn't propagate to config_db.json, I have no issues.

**Comment by jleveque** [comment]
@nikos-github: It will only propagate to config_db.json if you are loading your config from a minigraph file.

**Comment by nikos-github** [comment]
@jleveque Does the first install config of config_db.json load from minigraph?

**Comment by jleveque** [comment]
@nikos-github: No. Initial config is in JSON format.

File: src/sonic-config-engine/minigraph.py (hunk 2 above)
Comment (pra-moh) [reply src/sonic-config-engine/minigraph.py]: changed value to enabled as discussed

**Comment by jleveque** [comment]
@pra-moh: Please resolve new conflicts.

File: src/sonic-config-engine/minigraph.py [hunk 3]
```diff
@@ -728,8 +728,12 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['DHCP_SERVER'] = dict((item, {}) for item in dhcp_servers)
     results['NTP_SERVER'] = dict((item, {}) for item in ntp_servers)
     results['TACPLUS_SERVER'] = dict((item, {'priority': '1', 'tcp_port': '49'}) for item in tacacs_servers)
-
     results['ACL_TABLE'] = acls
+    results['FEATURES'] = {
```
Comment (jleveque) [inline src/sonic-config-engine/minigraph.py]: Naming convention for tables in SONiC DB is singular form. Can you change this to `FEATURE`?

Comment (pra-moh) [reply src/sonic-config-engine/minigraph.py]: my bad updated

**Pull request merged**